config_vsc.py

```diff
@@ -27,6 +27,11 @@
 # vsc group
 kul_account_string_tier2 = '-A lpt2_vsc_test_suite'
 
+# By default, not all installed modules are visible on the genius cluster
```

Comment:
why are the more recent modules not visible by default?

Reply:
Because somebody at some point decided it would be that way. Maybe it was done to avoid mixing of toolchain versions.